docker-compose-circle.yml

```diff
@@ -0,0 +1,90 @@
+version: '2'
+services:
+  persistent_db:
+    image: postgres:9.4
+    volumes_from:
+        - container:holder
```

Comment:
Not for this round, but if the volumes are the only significant change, what do you think about replacing them in a `sed` command or similar? That way we wouldn't need to maintain two configurations.